.github/workflows/test.yml

```
@@ -0,0 +1,92 @@
+        catch {
+          Write-Host "⚠️ Rust not found, will install"
+          "rust_installed=false" | Out-File -FilePath $env:GITHUB_OUTPUT -Append -Encoding utf8
+        }
+    
+    - name: Set up Rust
+      if: steps.rust-check.outputs.rust_installed != 'true'
+      uses: actions-rust-lang/setup-rust-toolchain@v1
+      with:
+        toolchain: stable
+        cache: false
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)